packages/discovery-provider/src/trending_strategies/trending_strategy_factory.py

```diff
@@ -21,6 +24,7 @@ def __init__(self):
         self.strategies = {
             TrendingType.TRACKS: {
                 TrendingVersion.pnagD: TrendingTracksStrategypnagD(),
+                TrendingVersion.AnlGe: TrendingTracksStrategyAnlGe(),
```

Comment:
I don't _think_ we need to add it to the underground section below since it will pull from the same results generated here. But I am not a full 100% sure.